medusa/storage/aws_s3_storage/awscli.py

```diff
@@ -73,7 +74,8 @@ def cp_upload(self, *, srcs, bucket_name, dest, max_retries=5):
         awscli_output = "/tmp/awscli_{0}.output".format(job_id)
         objects = []
         for src in srcs:
-            cmd = [self._aws_cli_path, "s3", "cp", str(src), "s3://{}/{}".format(bucket_name, dest)]
+            cmd = [self._aws_cli_path, "s3", "cp", "--acl", self.canned_acl,
```

Comment:
Would this be backwards compatible with the current behavior if set to `private` by default?
I seem to understand that it would, but want to make sure of it.

Reply:
Also, we're moving to a consolidated set of classes for all S3 compatible backends in #272. I'm wondering how this would affect other storage providers than AWS.
I'd be in favor of making this flag added to the aws command line only if `canned_acl` is set to anything else but `private` (or maybe we could leave it unset by default).  
@burmanm, wdyt?

Reply:
Yeah, I only added parameters to the command line if they differ from the default.

Reply:
@adejanovski according to aws s3 documentation, private is the default ACL (https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) which keeping it as a default option if is not set shouldn't change the behavior.